src/angular/styles/_typography.scss

```diff
@@ -880,54 +880,62 @@ fieldset,
       transform: scale(0.65);
       background-color: $sbbColorRed;
 
-      @include sbbLean {
+      .sbb-selection-item.sbb-lean & {
         background-color: $sbbColorIron;
       }
     }
   }
 }
 
-.sbb-selection-checked .sbb-selection-container {
-  & > .sbb-selection-container-checked {
-    display: block;
-  }
+.sbb-selection-checked {
```

Comment:
Is this not too component related and would better to be moved to the components css?